src/main/io/vtx_smartaudio.c

```diff
@@ -949,7 +952,7 @@ static void vtxSASetPitMode(vtxDevice_t *vtxDevice, uint8_t onoff)
         newMode |= SA_MODE_SET_OUT_RANGE_PITMODE;
     }
 
-    if (saDevice.mode & SA_MODE_GET_IN_RANGE_PITMODE || !(saDevice.mode & SA_MODE_GET_OUT_RANGE_PITMODE)) {
+    if (saDevice.mode & SA_MODE_GET_IN_RANGE_PITMODE || (onoff && !(saDevice.mode & SA_MODE_GET_OUT_RANGE_PITMODE))) {
```

Comment:
More coding style, sorry: Opening braces go onto the same line as the statement, unless it's a function body.

Reply:
Not exactly sure which braces you are talking about here, but i see that this isn't exactly readable. Ill try to make it a bit nicer.